crates/sui-indexer-alt-e2e-tests/src/lib.rs

```diff
@@ -116,7 +113,12 @@ pub struct OffchainCluster {
     /// Hold on to the temporary directory where the consistent store writes its data, so it
     /// doesn't get cleaned up until the cluster is stopped.
     #[allow(unused)]
-    dir: TempDir,
+    rocksdb_temp_dir: TempDir,
+
+    /// Hold on to the temporary directory where the consistent store writes its data, so it
+    /// doesn't get cleaned up until the cluster is stopped.
+    #[allow(unused)]
+    ingestion_temp_dir: Option<TempDir>,
```

Comment:
The OffchainCluster should not be responsible for this, because it can be configured to ingest over other sources. It's the responsibility of whatever's using the offchain cluster to set this up, and then configure it using `ClientArgs`.

Reply:
Alternatively (and maybe a better option), is to make it `OffchainCluster`'s responsibility to set-up `ClientArgs` (so remove that from the config), and then it's fine for it to also be responsible for its own temp directory, because you can't configure it to ingest any other way.